api-reference/10 UI Components/dxTabs/dxTabs.md

```diff
@@ -21,8 +21,8 @@ dx.web.js, dx.all.js
 #include common-demobutton with {
```

Comment:
I have a comment regarding line 9 above.

Line 9:
Can we update this line to the following?

`The Tabs component is a tab strip used to switch between pages or views. This UI component is included in the [TabPanel](/api-reference/10%20UI%20Components/dxTabPanel '/Documentation/ApiReference/UI_Components/dxTabPanel/') UI component, but you can use Tabs separately as well.`

Reply:
Noted